tox.ini

```diff
@@ -16,10 +16,11 @@ extras = dev
 deps =
     pytest
     pytest-cov
+    pytest-xdist
     coverage
     docutils
 commands =
-    pytest --basetemp="{envtmpdir}" {posargs:--color=yes --cov=dissect --cov-report=term-missing -v tests}
+    pytest --basetemp="{envtmpdir}" {posargs:--color=yes --cov=dissect --cov-report=term-missing -n 4 -v tests}
```

Comment:
Would `-n auto` also keep CI/CD happy? GitHub appears to have 4 cores: https://docs.github.com/en/actions/using-github-hosted-runners/using-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories

Reply:
Looks like you can set `PYTEST_XDIST_AUTO_NUM_WORKERS`: https://pytest-xdist.readthedocs.io/en/stable/distribution.html